spec/order_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'order'
+
+describe Order do
+  
+  # let(:message) {:order.confirm}
+  
+  describe '#add' do
+    it { is_expected.to respond_to(:add) }
+
+    it 'increases the order_list by 1' do
+      expect { subject.add("Kim Chi") }.to change { subject.order_list.count }.by(1) 
+    end 
+    # build a test which increases the 
```

Comment:
thanks for including comments on your what you next plan to add to the code and tests